RELEASE-NOTES.md

```diff
@@ -1,5 +1,71 @@
 
+## Release 2.7.2
+
+### Extensions to the new pausing feature!
+
+Pausing flow runs with Prefect have become much more powerful!
+
+Flow runs can now be paused without blocking the flow run process. This feature can free up agents to run other flows while waiting for approval. In order to use rescheduling pauses, flows must be deployed and results must be configured with the `persist_result` option.
+
+```
+from prefect import task, flow, pause_flow_run
+
+@task(persist_result=True)
+async def marvin_setup():
+    return "a raft of ducks walk into a bar..."
+
+@task(persist_result=True)
+async def marvin_punchline():
+    return "it's a wonder none of them ducked!"
+
+@flow(persist_result=True)
+async def inspiring_joke():
+    await marvin_setup()
+    await pause_flow_run(timeout=600, reschedule=True)  # pauses for 10 minutes
+    await marvin_punchline()
+```
+
+If set up as a deployment, running this flow will set up a joke, then pause and leave execution until it is resumed. Once resumed either with the `resume_flow_run` utility or the Prefect UI, the flow will be rescheduled and deliver the punchline.
+
+Deployed flows can also be paused outside of the flow process!
+
+The UI now features a **Pause** button on deployed flows that will stop execution at the beginning of the **next** task that runs. Any currently running tasks will be allowed to complete. Resuming this flow will reschedule it.
+
+Additionally, the `pause_flow_run` utility accepts an optional `flow_run_id` argument. When supplied, the specified flow run will be paused as long as it is still running and there are tasks remaining to run.
+
+
+See https://github.com/PrefectHQ/prefect/pull/7738 and https://github.com/PrefectHQ/prefect/pull/7863 for details
+
+### Enhancements
+- Improve Prefect import time by deferring imports — https://github.com/PrefectHQ/prefect/pull/7836
+- Add Opsgenie notification block — https://github.com/PrefectHQ/prefect/pull/7778
+- Add individual concurrency limit page with active runs list — https://github.com/PrefectHQ/prefect/pull/7848
+- Add `PREFECT_KUBERNETES_CLUSTER_UID` to allow bypass of `kube-system` namespace read — https://github.com/PrefectHQ/prefect/pull/7864
+- Refactor `pause_flow_run` for consistency with engine state handling — https://github.com/PrefectHQ/prefect/pull/7857
+- API: Allow `reject_transition` to return current state — https://github.com/PrefectHQ/prefect/pull/7830
+
+### Fixes
+- Fix bug where agent concurrency slots may not be released — https://github.com/PrefectHQ/prefect/pull/7845
+- Fix circular imports in the `orchestration` module — https://github.com/PrefectHQ/prefect/pull/7883
+- Fix deployments builds with scripts that contain flow calls - https://github.com/PrefectHQ/prefect/pull/7817
+- Fix path argument behavior in `LocalFileSystem` block - https://github.com/PrefectHQ/prefect/pull/7891
+
+### Documentation
+- Add CSS for beta and experimental badges — https://github.com/PrefectHQ/prefect/pull/7831
```

Comment:
```suggestion
```

Reply:
This is kind of inside baseball and I don't think it rates inclusion in the release notes, but open to other opinions.